examples/Simulated_ackerman_car/simulated_ackerman.cc

```diff
@@ -0,0 +1,57 @@
+// BoB robotics includes
+#include "common/main.h"
+#include "robots/simulatedAckermanCar.h"
+#include "robots/car_display.h"
+
+// Third-party includes
+#include "third_party/units.h"
+
+// Standard C++ includes
+#include <chrono>
+
+#include <iostream>
+#include <thread>
+
+using namespace BoBRobotics;
+using namespace std::literals;
+using namespace units::literals;
+
+int
+bob_main(int, char **)
+{
+    //Robots::SimulatedTank<> robot(1.4_mps, 500_mm); // Tank agent
+    Robots::SimulatedAckermanCar<> car(1.4_mps, 500_mm);
+    Robots::CarDisplay display(10.2_m,160_mm);                     // For displaying the agent
+
+    auto mmps = 1_mps;
+    units::angle::degree_t deg = units::angle::degree_t(20);
+    while(display.isOpen()) {
+
+        
+        car.move(mmps, deg);
+        auto key = display.runGUI(car.getPose());
+        
+        
+        if (key.first == SDLK_UP) {
```

Comment:
I'd do this as a ``switch`` block, because only one of these if-statements are going to be true.

Reply:
:+1:  (and that is **with** my hatred of switch statements)